info/v2/container.go

```diff
@@ -0,0 +1,97 @@
+// Copyright 2015 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package v2
+
+import (
+	"time"
+)
+
+type CpuSpec struct {
+	// Requested cpu shares. Default is 1024.
+	Limit uint64 `json:"limit"`
+	// Requested cpu hard limit. Default is unlimited (0)
+	MaxLimit uint64 `json:"max_limit"`
+	// Cpu affinity mask.
+	Mask string `json:"mask,omitempty"`
```

Comment:
SGTM, can we add a TODO to write a small library to do this? :) We already have it, we just need to export it here.